azurecompute-arm/src/test/java/org/jclouds/azurecompute/arm/compute/AzureComputeServiceLiveTest.java

```diff
@@ -56,18 +90,73 @@ protected ProviderMetadata createProviderMetadata() {
+      properties.setProperty(TIMEOUT_NODE_SUSPENDED, scriptTimeout + "");
+      properties.put(RESOURCE_GROUP_NAME, "a2");
+//      properties.put("jclouds.max-retries", 5);
+//      properties.put("jclouds.retries-delay-start", 5000L);
+
+      //properties.put(RESOURCE_GROUP_NAME, azureGroup);
 
       AzureLiveTestUtils.defaultProperties(properties);
       checkNotNull(setIfTestSystemPropertyPresent(properties, "oauth.endpoint"), "test.oauth.endpoint");
 
       return properties;
 
    }
+
+   //   @Override
+   protected Template refreshTemplate() {
+      return this.template = addRunScriptToTemplateWithDelay(this.buildTemplate(this.client.templateBuilder()));
+   }
+
+   protected static Template addRunScriptToTemplateWithDelay(Template template) {
+      template.getOptions().runScript(Statements.newStatementList(new Statement[]{AdminAccess.standard(), Statements.exec("sleep 50"), InstallJDK.fromOpenJDK()}));
+      return template;
+   }
+
+   @Override
+   @Test(
+           enabled = false
+   )
+   public void weCanCancelTasks(NodeMetadata node) throws InterruptedException, ExecutionException {
+      return;
+   }
+
+   @Override
+   protected void createAndRunAServiceInGroup(String group) throws RunNodesException {
+      ImmutableMap userMetadata = ImmutableMap.of("test", group);
+      ImmutableSet tags = ImmutableSet.of(group);
+      Stopwatch watch = Stopwatch.createStarted();
+      this.template = this.buildTemplate(this.client.templateBuilder());
+      this.template.getOptions().inboundPorts(new int[]{22, 8080}).blockOnPort(22, 300).userMetadata(userMetadata).tags(tags);
+      NodeMetadata node = (NodeMetadata) Iterables.getOnlyElement(this.client.createNodesInGroup(group, 1, this.template));
+      long createSeconds = watch.elapsed(TimeUnit.SECONDS);
+      String nodeId = node.getId();
+      this.checkUserMetadataContains(node, userMetadata);
+      this.checkTagsInNodeEquals(node, tags);
+      Logger.getAnonymousLogger().info(String.format("<< available node(%s) os(%s) in %ss", new Object[]{node.getId(), node.getOperatingSystem(), Long.valueOf(createSeconds)}));
+      watch.reset().start();
+      this.client.runScriptOnNode(nodeId, new StatementList(Statements.exec("sleep 50"), JettyStatements.install()), nameTask("configure-jetty"));
+      long configureSeconds = watch.elapsed(TimeUnit.SECONDS);
+      Logger.getAnonymousLogger().info(String.format("<< configured node(%s) with %s and jetty %s in %ss", new Object[]{nodeId, this.exec(nodeId, "java -fullversion"), this.exec(nodeId, JettyStatements.version()), Long.valueOf(configureSeconds)}));
+      this.trackAvailabilityOfProcessOnNode(JettyStatements.start(), "start jetty", node);
+      this.client.runScriptOnNode(nodeId, JettyStatements.stop(), org.jclouds.compute.options.TemplateOptions.Builder.runAsRoot(false).wrapInInitScript(false));
+      this.trackAvailabilityOfProcessOnNode(JettyStatements.start(), "start jetty", node);
+   }
```

Comment:
@nacx I think we need to override the entire `createAndRunAServiceInGroup` block since we need to add the `sleep 50` here and later we still need to add `ScriptBuilder.waitForLockFile` here.